tests/testthat/test-render.R

```diff
@@ -0,0 +1,193 @@
+testthat::describe("render() accepts", {
```

Comment:
These create the `report.Rmd` file on the directory, should we pass `keep_rmd = FALSE` or just change to a temporary working directory?

If I remember correctly CRAN is not fond of files being generated while tests are running.

Reply:
Thanks for a suggestion. I've implemented slightly different solution than `withr::local_tempdir/local_dir` as it needs a wrapping in additional `({ })` which become little too much for my eyes - `describe({ it({ local_tempdir({ ... })})})`. Look at the `with_temp_wd` on top of the testing file and let me know what do you think.